### ISSUE TITLE
Postgress16 image not working

### ISSUE THREAD

**Issue by SirOch27** [abesnier/docker-guacamole#40]
Hello,

I tried to upgrade from github-pg15 to github-pg16, but it failed.
In the logs, there is an error with S6 and postgressql:

s6-applyuidgid: fatal: unable to exec /usr/lib/postgresql/15/bin/postgres: No such file or directory
/var/run/postgresql:5432 - no response
Waiting for postgres to come up...****

Can you have a look please ?
I tried the other pg16 releases, and same error for all of them.

Regards
David

**Comment by abesnier** [comment]
You are right, I forgot to apply some changes. I am rebuilding, new images should be available in a few hours.

**Comment by abesnier** [comment]
Images updated, I tried a fresh `github-pg16` and it worked fine.

If you are using an existing image with PG 13/14 or 15, don't forget that PostgreSQL databases are not backwards compatible, and need to be exported in the previous version, then reimported in the new version. https://github.com/abesnier/docker-guacamole/blob/master/UPGRADE.md

**Comment by SirOch27** [comment]
Hello,

It works now, thank you ;-)
Yes i exported my DB15 before to upgrade

David

**Comment by abesnier** [comment]
Closed as per user feedback
